Meta/gn/secondary/Userland/Libraries/LibWeb/CSS/StyleValues/BUILD.gn

```diff
@@ -8,7 +8,12 @@ source_set("StyleValues") {
     "BasicShapeStyleValue.cpp",
     "BorderRadiusStyleValue.cpp",
     "CSSColorValue.cpp",
+    "CSSHSL.cpp",
```

Comment:
Re css-color-functions: In Serenity that's still a reftest (instead of a screenshot test), and it's easy to see what changed there: https://github.com/SerenityOS/serenity/pull/25099/commits/d32c3f44a0fb418d36f536a582b6f8d675087d2d#diff-0111303e7ec1a9d7c5643a07175aa7ddaf04750942bf9ce00b1a21088ebaf0ddL20

So it's nice that 50% and 0.5 now both have the same output, but according to comments on that file they're now both wrong, while before the percentage value was right :)

Reply:
Yeah I didn't delve into the maths to figure out what was up there. :sweat_smile: